test/resources/AdaptorHelperFunctions.sol

```diff
@@ -287,6 +294,65 @@ contract AdaptorHelperFunctions {
             );
     }
 
+    // ========================================= Morpho Blue FUNCTIONS =========================================
+
+    // MorphoBlueSupplyAdaptor Functions
+    function _createBytesDataToLendOnMorphoBlue(
+        MarketParams memory _market,
+        uint256 _assets
+    ) internal pure returns (bytes memory) {
+        return abi.encodeWithSelector(MorphoBlueSupplyAdaptor.lendToMorphoBlue.selector, _market, _assets);
+    }
+
+    function _createBytesDataToWithdrawFromMorphoBlue(
+        MarketParams memory _market,
+        uint256 _assets
+    ) internal pure returns (bytes memory) {
+        return abi.encodeWithSelector(MorphoBlueSupplyAdaptor.withdrawFromMorphoBlue.selector, _market, _assets);
+    }
+
+    function _createBytesDataToAccrueInterestToMorphoBlueSupplyAdaptor(
+        MarketParams memory _market
+    ) internal pure returns (bytes memory) {
+        return abi.encodeWithSelector(MorphoBlueSupplyAdaptor.accrueInterest.selector, _market);
+    }
```

Comment:
What's the difference between this one and `_createBytesDataToAccrueInterestToMorphoBlue`?

Reply:
We have accrue interest in all the morpho blue adaptors just to make it easier for strategists, so they can call the function if need be without needing to switch a different adaptor, then go back to the adaptor they want to use.